CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.18.0] - 2021-12-08
```

Comment:
Not sure why we need to release this as a breaking change. Furthermore, please revert the changes of the example/with-* apps to not use 0.18.0 for auth-react since that is something we will update after release of passwordless.

Reply:
@porcellus did you miss this comment?

Reply:
I did sorry. I'll revert them and add it as a patch version. Although according to semver we are supposed to be increasing the minor version, I guess we are not breaking anything and we are in the 0.X territory anyway.

Reply:
Actually.. This is a new "big" feature. So even if it's not breaking, perhaps it makes sense to keep it this way..?

Reply:
I think it makes more sense that way. Should I move it to 0.19.0?

Reply:
It should be moved to 0.18.0 since the latest released is 0.17.X